webapp/src/reducers/index.js

```diff
@@ -193,6 +193,39 @@ const channelSubscriptions = (state = {}, action) => {
     }
 };
 
+const getIssueByKey = (state = {}, action) => {
+    switch (action.type) {
+    case ActionTypes.RECEIVED_JIRA_TICKETS : {
+        const assignee = action.data && action.data.fields && action.data.fields.assignee ? action.data.fields.assignee : null;
+        const ticketData = action.data;
+        const ticketDetails = {
+            assigneeName: assignee && assignee.displayName ? assignee.displayName : '',
+            assigneeAvatar: assignee && assignee.avatarUrls && assignee.avatarUrls['48x48'] ? assignee.avatarUrls['48x48'] : '',
+            labels: ticketData.fields.labels,
+            description: ticketData.fields.description,
+            summary: ticketData.fields.summary,
+            ticketId: ticketData.key,
+            jiraIcon: ticketData.fields.project.avatarUrls['48x48'],
+            versions: ticketData.fields.versions.lenght > 0 ? ticketData.fields.versions.versions[0] : '',
```

Comment:
```suggestion
            versions: ticketData.fields.versions.length ? ticketData.fields.versions.versions[0] : '',
```

Reply:
also, do we need to explicitly check length here?

Reply:
yes

Reply:
Not sure what you are asking here. But I think I made the change you are suggesting